### ISSUE TITLE
Complaints from Windows 7 users

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#161]
Apparently windows 7 doesn't work on the site, issues don't show up.

**Comment by benjaffe** [comment]
Any idea which browser(s)?

**Comment by nickoneill** [comment]
Didn't say, but I'm guessing the default. I'll ask around at work today to see if I can find a win 7 machine.

**Comment by benjaffe** [comment]
I just used BrowserStack to try with Windows 7 and IE 11. Confirmed the issue... the left bar is absent.

![image](https://cloud.githubusercontent.com/assets/573204/22656101/0e7a1622-ec47-11e6-86a8-aaf28d74db31.png)

Works on Windows 7 and latest Firefox/Chrome

**Comment by nickoneill** [comment]
Wow, I had no idea this was a thing. Guess I've been gone from frontend eng for too long? Does it give you a console too?

**Comment by nickoneill** [comment]
I didn't see the same behavior on browserstack: my list of issues showed up, but clicking on them does nothing. Still investigating.

**Comment by benjaffe** [comment]
Hmm, to clarify, I loaded with a different browser first, then switched to IE 11. I wonder if there's some caching going on when loading from localhost with their extension, since the list showed up (though the clicking was broken for me)

When I entirely stopped and restarted the server and the session, then tried going straight for Windows 7 and IE 11, that's when I saw the screenshot above.

I can't test properly, so I guess my half-repro case is the best I can do right now. :/

**Comment by oderby** [comment]
I have a Windows 7 desktop at home. Tried out the site, reporting what I noticed:

1. Can confirm Chrome on Windows 7 works fine
2. on IE 11, on fresh load of page (never loaded before), IE prompts me to let the site use location. If I deny, prompt disappears but nothing happens. No errors in the console or anything, and I didn't notice anything change on the page (no zipcode prompt, no issues appeared, etc.)
3. If I do let the site access my location (after initially refusing, reloading, refusing again and then reloading), I get an error in the console about failing to get ip address, and then I have to put in my zipcode (I admit I did several things, so would be worth re-testing this behavior).
4. After putting in zip code, issues appear, but I perceive the same issue as described above, where clicking on an issue changes the Window.location, but nothing on the page updates. If I refresh, the issue does display.
5. To attempt some initial triage, I opened the dev tools in IE 11 and Chrome and played with some breakpoints. I got as far as to confirm that the two browsers differ in behavior at https://github.com/5calls/5calls/blob/master/static/js/components/content.js#L8 - in IE11, `state.location.params` is an empty object (so the property `issueid` is `undefined`), while in Chrome `state.location.params.issueid` equals the issue id in the link hash. 

Unfortunately, my browser crashed before I could diagnose further. My hunch is that there are actually two issues in this ticket - one around issues not showing up at all on IE 11 initially (perhaps related to availability of ip-based geolocation?), and the other about issue navigation not working properly on IE 11 at all. Happy to test further tomorrow if you have other ideas...
